### ISSUE TITLE
2.12.0-alpha1 pre-release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3799]
Release project board: https://github.com/orgs/az-digital/projects/82

- #3406 
- #3638
 - #3631 
- #3680
- #3677 
- #3610 
- #3662 
- #3585
- #3739 
- #3771
- #3808 
- #3219 
- #3696 
- #3814
- #2347 
- #3764 
- #3786 

## Release notes included in PRs

From #3610:
```
### Visual changes to News items
AZ News content type displays are changed to have the image on the left instead of right of the content.

AZ News view displays have been changed. The former "Small Row View" display has changed its name to "Sidebar Row View". A new view display called "Small Row View" has been added to show the new "Small Row" display of AZ News content.
```

From #3662:
```
### Better exposed filters dependencies no longer needed
This release updates [Better Exposed Filters](https://www.drupal.org/project/better_exposed_filters) from 6.0.6 to the 7.x major version. The [jquery_ui_slider](https://www.drupal.org/project/jquery_ui_slider) and [jquery-ui-touch-punch](https://www.drupal.org/project/jquery_ui_touch_punch) modules are no longer required for Better Exposed Filters or for anything else in Quickstart. These two modules have been deprecated as Quickstart dependencies and will be removed in a future minor release of Quickstart. Site owners should uninstall these modules or add them to their site-specific composer.json files to continue using them.
```

From #3808:
```
### Deprecated variable in az-accordion.html.twig template
The `aria_controls` variable used in the `az_accordion` theme implementation and `az-accordion.html.twig` template has been deprecated as of 2.12.0 and will be removed in 2.13.0.  Any subthemes that override this template will need to replace the `aria_controls` variable with the `accordion_item_id` variable.  See https://github.com/az-digital/az_quickstart/issues/3807.

```

From #3219:
```
### CKEditor Anchor Link Plugin added
Users with the content editor role can now easily create anchor links in text areas using the Standard text formatter.
```

From #3696:
```
### filter_autop removed from Standard Text format
This release disables the `Convert line breaks into HTML (i.e. <br> and <p>`) filter for the Standard text format. This change resolves issues with the appearance and functionality of multiple elements in Quickstart. However, while most existing content should be unaffected, this change may result in some minor spacing changes, where extra blank lines may be removed or added. We may consider enabling this filter again in the future if [this Drupal core](https://www.drupal.org/project/drupal/issues/3075527) issue regarding the filter is resolved.
```

From #3814:
```
This change makes the Card Link Title field required if the Card Link URL field is populated.
```

From #3764:
```
 A new Profile Editor role has been added to Quickstart. This role gains permission to edit Person nodes where the user is assigned as the author, or Person nodes that have the same NetID as the user's login.
 
 The role also has the ability to change the order and visibility of publications shown on the Publication block on their Person page.
```

From #3786:
```
### Change to summary fields
News, Events, and Pages with a populated summary field values will notice a change if the content within the field has a link or email. These will no longer be automatically linked and will rendered as plain text.
```

**Comment by joeparsons** [comment]
_This is a pre-release for the next minor version (feature release) of Arizona Quickstart 2. Pre-releases are not supported for production sites but are intended for widespread testing in preparation for the upcoming stable release._

_**Quickstart 2.10.x will no longer receive security support. Sites on Quickstart 2.10.x or earlier should upgrade to a supported release as soon as possible.** See our [release policy](https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#release-overview) for more information._

_For more information about the changes in this release, please see the [project page](https://github.com/orgs/az-digital/projects/82) and the [milestone](https://github.com/az-digital/az_quickstart/milestone/22)._

## Important Notes

### How to update
Refer to [Updating your Quickstart website](https://github.com/az-digital/az_quickstart/wiki/Maintaining-Quickstart-2-Sites#updating-your-quickstart-website) on the wiki for more information.

## New changes (not included in 2.11.x)

### Bug Fixes:
- Closes #3569, #3689 Disable automatic paragraph filter for the Standard text format (#3696)
- Closes #3793: Improve accessibility of card link titles (#3814)

### General Enhancements:
- Fixes #3619 Add Documentation for az-core-distribution-config (#3638)
- Closes #3576 Add new demo AZ News node with embedded media and extra paragraphs (#3631)
- Fixes #3678 Add Generator Metatag (#3680)
- Closes #3086 Adding demo content for publications (#3585)
- Closes #2924 Add suffix field to person + demo content (#3739)
- Closes #3807: Deprecate aria_controls variable in az_accordion theme implementation and template. (#3808)
- Closes #3217: Add anchor link plugin for CKEditor5. (#3219)
- Closes #2345: Add automated test to verify cache invalidation is working. (#2347)
- Fixes #3553, Fixes #3790 AZ Person manageable using CAS account login, Dedicated Permission for Publication Curation (#3764)

### Contrib Module Updates:
- Update drupal/workbench_access requirement from 2.0.1 to 2.0.3 (#3406)
- Update drupal/better_exposed_filters requirement from 6.0.6 to 7.0.1 (#3662)
- Update drupal/better_exposed_filters requirement from 7.0.1 to 7.0.2 (#3771)
 
### Visual Changes:
- Fixes #3676 Course Display Changes (#3677)
- Closes #3590: Change News preview layouts (#3610)
- Closes #2929: Strip links from summary field(s). (#3786)

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.12.0-alpha1
